src/WebApi.OutputCache.V2/CacheOutputAttribute.cs

```diff
@@ -242,12 +255,12 @@ public override void OnActionExecuting(HttpActionContext actionContext)
                         _webApiCache.Add(baseKey, string.Empty, cacheTime.AbsoluteExpiration);
                         _webApiCache.Add(cachekey, content, cacheTime.AbsoluteExpiration, baseKey);
```

Comment:
Wouldn't this effectively invalidate the cache for _all_ clients anyways since we are adding(replacing) the cache entry for the same exact cache key that we are explicitly removing in Line 234 ? Since the cache is public and not a per client private cache, do you see any use cases wherein it's harmful to have the cache invalidated for a specific key "globally", if requested so by one client by specifying "cache-control:no-cache"